dashboard/analyse_dashboard/analyse/functions.py

```diff
@@ -1464,16 +1480,29 @@ def wait_bin_cluster_redenna(toestemming_df):
     return wait_bin_cluster_redenna_df
 
 
-def quality_measures_by_project(df: pd.DataFrame):
+def calculate_projectindicators_tmobile(df: pd.DataFrame):
+    title = pd.DataFrame(index=['on_time', 'limited_time', 'late', 'before_order'],
+                         data=['Order op tijd', 'Order nog beperkte tijd', 'Order op tijd', ''],
+                         columns=['title'])
+    subtitle = pd.DataFrame(index=['on_time', 'limited_time', 'late', 'before_order'],
+                            data=['< 8 weken', '> 8 weken < 12 weken', '> 12 weken', ''],
+                            columns=['subtitle'])
+    font_color = pd.DataFrame(index=['on_time', 'limited_time', 'late', 'before_order'],
+                              data=['green', 'orange', 'red', ''],
+                              columns=['font_color'])
+    id_ = pd.DataFrame(index=['on_time', 'limited_time', 'late', 'before_order'],
```

Comment:
Adding the ids here is redundant. This information can be constructed from the index and client in the frontend. This is more limiting in what the eventual id will be.